RecoHI/HiJetAlgos/python/HiRecoPFJets_cff.py

```diff
@@ -96,13 +93,13 @@
     rhoFlowFitParams = cms.InputTag('hiFJRhoFlowModulation', 'rhoFlowFitParams'),
     jetCollInstanceName = cms.string("pfParticlesCs"),
 )
-akCs4PFJets.src           = cms.InputTag('particleFlow')
-akCs4PFJets.doAreaFastjet = cms.bool(True)
-akCs4PFJets.jetPtMin      = cms.double(0.0)
+akCs4PFJets.src               = 'particleFlow'
+akCs4PFJets.doAreaFastjet     = True
+akCs4PFJets.jetPtMin          = 0.0
 akCs4PFJets.useExplicitGhosts = cms.bool(True)
```

Comment:
have you tried ` akCs4PFJets.useExplicitGhosts = True `?

Reply:
> have you tried `akCs4PFJets.useExplicitGhosts = True`?

The `useExplicitGhosts` parameter is missing in the `akCs4PFJets` config, contrarily to the other ones there which can therefore get modified without  defining the type